languages.toml

```diff
@@ -121,6 +121,9 @@ scope = "source.json"
 injection-regex = "json"
 file-types = ["json"]
 roots = []
+language-server = { command = "vscode-json-language-server", args = ["--stdio"] }
```

Comment:
How is this normally installed? I see the [vscode-json-languageserver](https://www.npmjs.com/package/vscode-json-languageserver) but the binary is not the same name

Reply:
@EricCrosson Did you tried to install it? Package name can be different than package path. I installed from AUR, it work on `vscode-json-language-server` path.

Reply:
it's available under another npm package: https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#jsonls

Reply:
@EpsilonKu I did :slightly_smiling_face: no dice

Thanks @the-mikedavis!